src/assets/_scss/extends/_slider.scss

```diff
@@ -0,0 +1,97 @@
+%slider {
+  -webkit-appearance: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  &:focus {
+    outline: none;
+
+    &::-webkit-slider-runnable-track {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-webkit-slider-thumb {
+      border: 1px solid darken($color-gray-cool-light, 10);
+    }
+
+    &::-ms-fill-lower {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-ms-fill-upper {
+      background: lighten($color-gray-cool-light, 3);
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
+    background: lighten($color-gray-cool-light, 3);
+    border: 1px solid darken($color-gray-cool-light, 15);
+    border-radius: 0.5em;
+    cursor: pointer;
+    height: 0.5em;
+    width: 100%;
+  }
+
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    background: $color-white;
+    border: 1px solid darken($color-gray-cool-light, 10);
+    border-radius: 50%;
+    box-shadow: 0 2px 3px rgba($color-black, 0.4), 0 0 0 #0d0d0d;
+    cursor: pointer;
+    height: 1.5em;
+    margin-top: -0.65em;
+    width: 1.5em;
+  }
+
+  &::-moz-range-track {
+    background: lighten($color-gray-cool-light, 3);
+    border: 1px solid $color-gray-cool-light;
+    border-radius: 0.5em;
+    box-shadow: 0 0 1px rgba(0, 0, 0, 0), 0 0 0 rgba(13, 13, 13, 0);
+    cursor: pointer;
+    height: 7.6px;
+    width: 100%;
+  }
+
+  &::-moz-range-thumb {
+    border-radius: 50%;
+    background: $color-white;
+    border: 1px solid rgba(95, 95, 95, 0.4);
+    box-shadow: 0 4px 5px rgba($color-black, 0.4), 0 0 0 #0d0d0d;
+    cursor: pointer;
+    height: 1.5em;
+    width: 1.5em;
+  }
+
+  &::-ms-track {
+    height: 1.5em;
+    cursor: pointer;
+    background: transparent;
+    border-color: transparent;
+    color: transparent;
+    width: 100%;
+  }
+
+  &::-ms-fill-lower {
+    background: rgba(187, 187, 187, 0.73);
+    border-radius: 50%;
+    box-shadow: 0 0 1px rgba(0, 0, 0, 0), 0 0 0 rgba(13, 13, 13, 0);
+  }
+
+  &::-ms-fill-upper {
+    background: rgba(200, 200, 200, 0.73);
```

Comment:
Color literals like `rgba(200, 200, 200, 0.73)` should only be used in variable declarations; they should be referred to via variable everywhere else.